### PR TITLE
add more reveal features to slide demo

### PR DESCRIPTION
**Pull request by butlerx**
I wrote this for the my internal hackmd to show off reveal.js
[demo](https://md.redbrick.dcu.ie/p/slide-example)

### CODE REVIEW

**Comment by jackycute** [comment]
It looks great!

**Comment by jackycute** [comment]
@SISheogorath Could you help us review this PR?

File: public/docs/slide-example.md [hunk 1]
```diff
@@ -23,31 +28,68 @@ Is the divider of slides
 
 Is the divider of branches
 
+Use the _Space_ key to navigate through all slides.
+
 ----
 
 ### Second branch of first slide
```
Comment (SISheogorath) [inline public/docs/slide-example.md]: It should say "of the first slide".

Not only here, also in https://github.com/butlerx/hackmd/blame/50153c65c6ac37ff783b695f5c6bab2ac72ff517/public/docs/slide-example.md#L25

File: public/docs/slide-example.md [hunk 2]
```diff
@@ -23,31 +28,68 @@ Is the divider of slides
...
 
+Nested slides are useful for adding additional detail underneath a high level horizontal slide.
```
Comment (SISheogorath) [inline public/docs/slide-example.md]: Should be `high-level` instead of `high level`.

File: public/docs/slide-example.md [hunk 3]
```diff
@@ -23,31 +28,68 @@ Is the divider of slides
...
+Note:
+  This slide has fragments which are also stepped through in the notes window.
 
 ---
 
-## Second slide
+## Fragment Styles
 
-<!-- .slide: data-background="#1A237E" -->
+There's different types of fragments, like:
```
Comment (SISheogorath) [inline public/docs/slide-example.md]: Should be `There are`

File: public/docs/slide-example.md [hunk 4]
```diff
@@ -80,4 +122,142 @@ default/fast/slow
...
 
-# The End
\ No newline at end of file
+## Themes
+
+reveal.js comes with a few themes built in:  
+Black (default) - White - League - Sky - Beige - Simple  
+Serif - Blood - Night - Moon - Solarized
+
+It can be set in Yaml slideOptions
```
Comment (SISheogorath) [inline public/docs/slide-example.md]: Should be `YAML` as it is an abbreviation for "Yet Another Markup Language".

File: public/docs/slide-example.md [hunk 5]
```diff
@@ -80,4 +122,142 @@ default/fast/slow
...
+
+---
+
+## Intergalactic Interconnections
+
+You can link between slides internally, [like this](#/1/3).
+
+---
+
+## Speaker View
```
Comment (SISheogorath) [inline public/docs/slide-example.md]: As the official feature is called "Speaker Notes" I would also call this section "Speaker Notes". It makes it easier for the people who already use `reveal.js` to stay on topic.

**Comment by jackycute** [comment]
Thanks a lot @butlerx @SISheogorath

**Pull request merged**